docs-ref-conceptual/includes/cli-install-linux-apt.md

```diff
@@ -69,13 +69,21 @@ If you prefer a step-by-step installation process, complete the following steps
     sudo apt-get install azure-cli
     ```
 
-## Sign in to Azure with the Azure CLI
+## Install specific version
 
-Run the Azure CLI with the `az` command. To sign in, use the [az login](/cli/azure/reference-index#az_login) command.
+You must first configure `azure-cli` repository information as shown above.
 
-[!INCLUDE [interactive-login](interactive-login.md)]
+1. To view available versions:
 
-To learn more about different authentication methods, see [Sign in with Azure CLI](../authenticate-azure-cli.md).
```

Comment:
The "sign in" section is duplicated many times and is actually irrelevant to the topic of installation.